internal/kubernetes/operator/common/const.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
```

Comment:
👃 let's avoid generic package names they are a code smell, it seems like this constant is only used in one file, let's move it there

Reply:
Fixed, package has been renamed to `pointers`, the constant has been moved to `project` package